controllers/volumereplicationgroup_controller.go

```diff
@@ -429,6 +430,12 @@ func (r *VolumeReplicationGroupReconciler) Reconcile(ctx context.Context, req ct
 	v.ramenConfig = ramenConfig
 	adminNamespaceVRG := vrgInAdminNamespace(v.instance, v.ramenConfig)
 
+	if adminNamespaceVRG && !r.veleroCRsAreWatched {
+		return ctrl.Result{},
+			fmt.Errorf("VRG {%s/%s} with kube object protection doesn't work if velero/oadp is not installed. "+
+				"Please install velero/oadp and restart the operator", v.instance.Namespace, v.instance.Name)
+	}
```

Comment:
Can you share actual logs showing these errors? I think we will see lot of unwanted and unhelpful repeated messages with this. Each one will have unhelpful trackback.